orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -330,6 +379,60 @@ void addObjectMethodsForTraining(py::module& m) {
         }
         ORT_THROW_IF_ERROR(static_cast<TrainingSession*>(sess->GetSessionHandle())->SetStateTensors(state_tensors, strict));
       })
+      .def("load_model_opt_state", [](PyTrainingSession* sess, std::unordered_map<std::string, py::object>& model_state, std::unordered_map<std::string, std::unordered_map<std::string, py::object>>& opt_state, bool strict) {
```

Comment:
It seems like the ```model_state``` here is an ```unordered_map<string, py::object>```. Should it not be  ```unordered_map<string, unordered_map<string, py::object>>``` considering the below structure:
```
{
    'fp32':
    {
        model_weight: model_weight_tensor
    }
}
```

Reply:
You're right, I was thinking about only allowing passing in fp32 weights. but it needs to include both. Changed to `unordered_map<string, unordered_map<string, py::object>>`